documentation/guides/legacy-polaris-v8-public-api.scss

```diff
@@ -1347,7 +1347,7 @@ $nav-min-window: em(layout-width(page-with-nav));
     @include high-contrast-outline;
   }
 
-  &:focus {
+  &:focus-visible {
```

Comment:
This code doesn't need to be changed as it isn't published, it's just a reference to the legacy Sass styles.